.vib/airflow/goss/airflow.yaml

```
@@ -0,0 +1,39 @@
+command:
+  check-scheduler:
+    timeout: 30000
+    exec: airflow users list
+    exit-status: 0
+    stdout:
+      - No data found
+  check-subpackages:
+    # Check python packages that should have been installed
+    exec: . /opt/bitnami/airflow/venv/bin/activate && pip list
+    exit-status: 0
+    stdout:
+      # all_dbs does not show in the package list so we don't add it
+      - async
+      - amazon
+      - celery
+      - /cncf.kubernetes/
+      - docker
+      - dask
+      - elasticsearch
+      - ftp
+      - grpc
+      - hashicorp
+      - http
+      - ldap
+      - google
+      - /microsoft.azure/
+      - mysql
+      - postgres
+      - redis
+      - sendgrid
+      - sftp
+      - slack
+      - ssh
+      - statsd
+      - virtualenv
```

I would not check this whole list of modules, just a subset of the more relevant ones:

```suggestion
      - async
      - amazon
      - celery
      - /cncf.kubernetes/
      - docker
      - hashicorp
      - ldap
      - google
      - /microsoft.azure/
      - mysql
      - postgres
      - redis
```

---

I've also moved the list to the vars.yaml, based on earlier comments from other diffs.